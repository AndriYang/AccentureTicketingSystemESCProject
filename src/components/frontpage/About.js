import React, { Component } from 'react'
import { connect } from 'react-redux'

export const About = () => {

  return(
    <div className="card-panel">
      <h1 class="blue-text text-darken-2"> WHAT WE DO  </h1>
      <p>Accenture solves our clients toughest challenges by providing unmatched services in strategy, consulting, digital, technology and operations.</p>
      <p> We partner with more than three-quarters of the Fortune Global 500, driving innovation to improve the way the world works and lives.</p>
      <p> With expertise across more than 40 industries and all business functions, we deliver transformational outcomes for a demanding new digital world. </p>
    <div class="divider"></div>
    <div class="section">
      <h4 class="blue-text text-darken-2">Accenture Strategy</h4>
      <p>Shapes the future at the intersection of business and technology</p>
    </div>
    <div class="divider"></div>
    <div class="section">
      <h4 class="blue-text text-darken-2">Accenture Consulting</h4>
      <p>Transforms businesses through industry expertise and insights</p>
    </div>
    <div class="divider"></div>
    <div class="section">
      <h4 class="blue-text text-darken-2">Accenture Digital</h4>
      <p>Creates value through new experiences, new intelligence and new connections</p>
    </div>
    <div class="divider"></div>
    <div class="section">
      <h4 class="blue-text text-darken-2">Accenture Technology</h4>
      <p>Powers businesses with cutting-edge solutions using established and emerging technologies</p>
    </div>
    <div class="divider"></div>
    <div class="section">
      <h4 class="blue-text text-darken-2">Accenture Operation</h4>
      <p>Delivers outcomes through infrastructure, security, cloud and business process services</p>
    </div>
  </div>

  )
}

export default About
