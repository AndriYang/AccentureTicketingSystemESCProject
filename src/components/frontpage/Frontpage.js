import React, { Component } from 'react'
import { connect } from 'react-redux'
import { compose } from 'redux'
import { Redirect } from 'react-router-dom'
import { Link } from 'react-router-dom'


export const Frontpage = () => {

  return (
    <form >

      <div className="card-content">
        <div className="row ">
          <div class="col s12 m12 l12 offset-s6 center-align ">
            <Link to='/admin' id="admin" class="btn-large  purple darken-3"> Admin </Link>
          </div>
        </div>
          <div class="offset-s6 center-align z-depth-0">
            <Link to='/guesscustomerquery' id="customer" class="btn-large  purple darken-3"> Customer </Link>
          </div>

      </div>

    </form>

  )
}

export default Frontpage
