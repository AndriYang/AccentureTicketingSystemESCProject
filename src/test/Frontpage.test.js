import React from 'react';
import { shallow, mount, render } from 'enzyme';
import {Frontpage} from '../components/frontpage/Frontpage';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
import renderer from 'react-test-renderer';
import { Link } from 'react-router-dom';
import { create } from "react-test-renderer";

configure({ adapter: new Adapter() });
describe('Number of', ()=> {
  const wrapper = shallow(<Frontpage />);

  it('Link and its content', function() {
    expect(wrapper.find('Link').length).toBe(2);
    expect(wrapper.find('Link').at(1).text()).toBe('<Link />');
    expect(wrapper.find(Link).first().props().to).toBe('/admin');
    expect(wrapper.find(Link).at(1).props().to).toBe('/guesscustomerquery');
  });

  it('Link directed to', function() {
    expect(wrapper.find(Link).first().props().to).toBe('/admin');
    expect(wrapper.find(Link).at(1).props().to).toBe('/guesscustomerquery');
  });
})

describe("guesscustomerquery component", () => {
  test("it matches the snapshot", () => {
    const component = create(<guesscustomerquery />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
