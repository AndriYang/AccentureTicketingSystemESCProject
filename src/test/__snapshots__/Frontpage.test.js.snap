// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`guesscustomerquery component it matches the snapshot 1`] = `<guesscustomerquery />`;
